src/deluge/util/functions.h

```diff
@@ -215,8 +215,7 @@ void addAudio(StereoSample* inputBuffer, StereoSample* outputBuffer, int32_t num
 
 char const* getSourceDisplayNameForOLED(PatchSource s);
 char const* getPatchedParamDisplayName(int32_t p);
-char const* getUnpatchedParamDisplayName(int32_t p);
-char const* getGlobalEffectableParamDisplayName(int32_t p);
+char const* getParamDisplayName(Param::Kind kind, int32_t p);
```

Comment:
Can we call this function getUnpatchedParamDisplayName since they're all unpatched params right? Unpatched_Sound or Unpatched_Global.

Thx

Reply:
No, it also delegates to getPatchedParamDisplayName. The intention is to add the ability to get the name of any params, including Midi CCs for instance